### PR TITLE
updated dac2018

### PR DESCRIPTION
**Pull request by kentaroy47**
updated ml papers from dac 2018.
congrats on your isca!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -367,12 +367,12 @@ This is a collection of conference papers that interest me. The emphasis is focu
...
-- EVA^2: Exploiting Temporal Redundancy in Live Computer Vision. (Cornell)
-- Euphrates: Algorithm-SoC Co-Design for Low-Power Mobile Continuous Vision. (Rochester, Georgia Tech, ARM)
-- Feature-Driven and Spatially Folded Digital Neurons for Efficient Spiking Neural Network Simulations. (POSTECH/Berkeley, Seoul National)
-- Space-Time Algebra: A Model for Neocortical Computation. (Wisconsin)
-- Scaling Datacenter Accelerators With Compute-Reuse Architectures. (Princeton)
-- Enabling Scientific Computing on Memristive Accelerators. (Rochester)
+- **EVA^2: Exploiting Temporal Redundancy in Live Computer Vision.** (Cornell)
+- **Euphrates: Algorithm-SoC Co-Design for Low-Power Mobile Continuous Vision.** (Rochester, Georgia Tech, ARM)
+- **Feature-Driven and Spatially Folded Digital Neurons for Efficient Spiking Neural Network Simulations.** (POSTECH/Berkeley, Seoul National)
+- **Space-Time Algebra: A Model for Neocortical Computation.** (Wisconsin)
```
Comment (fengbintu) [inline README.md]: Thanks, Ken. I prefer unbold Line 373~375, as the bold font in this survey usually means they are quite related to deep learning accelerators, though I think the three papers are interesting.

File: README.md [hunk 2]
```diff
@@ -383,7 +383,16 @@ This is a collection of conference papers that interest me. The emphasis is focu
...
-- HyperPower: Power and Memory-Constrained Hyper-Parameter Optimization for Neural Networks. (CMU, Google)
+- **HyperPower: Power and Memory-Constrained Hyper-Parameter Optimization for Neural Networks.** (CMU, Google)
+
+### 2018 DAC
+- **DPS: Dynamic Precision Scaling for Stochastic Computing-Based Deep Neural Networks.** (UNIST)
+- **DyHard-DNN: Even More DNN Acceleration With Dynamic Hardware Reconfiguration** (Univ. of Virginia)
+- **Exploring the Programmability for Deep Learning Processors: from Architecture to Tensorization** (Univ. of Washington)
+- **LCP: Layer Clusters Paralleling Mapping Mechanism for Accelerating Inception and Residual Networks on FPGA** (THU)
+- **Ares: A Framework for Quantifying the Resilience of Deep Neural Networks** (Harvard)
+- **Loom: Exploiting Weight and Activation Precisions to Accelerate Convolutional Neural Networks** (Univ. Tronto)
```
Comment (fengbintu) [inline README.md]: Here may be Univ. of Toronto.

File: README.md [hunk 3]
```diff
@@ -383,7 +383,16 @@ This is a collection of conference papers that interest me. The emphasis is focu
    - *Decomposes convolution into multiple dense and zero kernels for sparsity savings.*
 - **Block Convolution: Towards Memory-Efficient Inference of Large-Scale CNNs on FPGA.** (CAS)
 - **moDNN: Memory Optimal DNN Training on GPUs.** (University of Notre Dame, CAS)
-- HyperPower: Power and Memory-Constrained Hyper-Parameter Optimization for Neural Networks. (CMU, Google)
+- **HyperPower: Power and Memory-Constrained Hyper-Parameter Optimization for Neural Networks.** (CMU, Google)
+
+### 2018 DAC
```
Comment (fengbintu) [inline README.md]: Many thanks, Ken. I intend to survey the DAC papers these days. Your contributions help us a lot. I find there are much more ml papers than the ones you list here. I guess you have picked the highest-quality papers from the proceedings. That's great!

**Comment by fengbintu** [comment]
Thanks, @kentaroy47 . You really help us a lot! Could you please check my comments and pull a new request?

File: README.md (hunk 3 above)
Comment (kentaroy47) [reply README.md]: I might have left out some.. It would be great if other can keep adding interesting papers! :)

**Pull request merged**